docs/install_guides/configurations-for-nvidia-gpu.md

```diff
@@ -0,0 +1,78 @@
+# Configurations for NVIDIA GPUs with OpenVINO™ {#openvino_docs_install_guides_configurations_for_nvidia_gpu}
+
+@sphinxdirective
+
+.. meta::
+   :description: Learn how to provide additional configuration for NVIDIA 
+                 GPUs to work with Intel® Distribution of OpenVINO™ toolkit 
+                 on your system.
+
+
+.. _nvidia gpu guide:
+
+OpenVINO™ NVIDIA GPU plugin enables deep neural networks inference on NVIDIA GPUs. 
+The plugin uses custom kernels and cuBLAS, cuDNN, cuTENSOR libraries as a backend 
+and requires prior configuration on the following supported operating systems:
+
+* Ubuntu 20.04 long-term support (LTS), 64-bit
+* Windows 10, 64-bit
+* Windows 11, 64-bit
+
+OpenVINO™ NVIDIA GPU plugin is not included in OpenVINO™ toolkit. If you want to use 
+the plugin, first you need to `install dependencies <install-dependencies>`__, and 
+then `build it from source code <#whats-next>`__.
```

Comment:
```suggestion
If you want to use the plugin, first you need to `install dependencies <install-dependencies>`__, and 
then `build it from source code <#whats-next>`__.
```

Reply:
We need to remove the whole paragraph, including referring to `build it from source code <#whats-next>`.